appserver/common/amx-jakartaee/pom.xml

```diff
@@ -17,7 +17,7 @@
 
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsdd">
```

Comment:
Oh god, really! Thanks, that must be another bug in Eclipse IDE, sad ☹️
I will fix it when I get to my laptop, and I am happy we agreed.

Reply:
FYI, tidy:1.2.0:pom does this transformation too.

Reply:
So it is really a bug in Eclipse ... now it swallowed the `"` character and then again duplicated the last one. Meh!!!

Reply:
Aha, the problem occurs when it is a subproject once imported to Eclipse IDE and once seen as a subfolder of another project. Eclipse now does the conversion twice and that has some unpredictable results. The workaround is to exclude duplicities, or to mark those subfolders manually as "derived".